dev/archery/archery/integration/datagen.py

```diff
@@ -609,9 +609,9 @@ def generate_column(self, size, name=None):
 
         sizes = self._random_sizes(size)
 
-        for i, nbytes in enumerate(sizes):
+        for i, np_nbytes in enumerate(sizes):
             if is_valid[i]:
-                values.append(random_bytes(nbytes))
+                values.append(random_bytes(int(np_nbytes)))
```

Comment:
Could you add a comment why we need this workaround and when we can remove this workaround?